README.md

```diff
@@ -47,20 +47,15 @@
 
 ### Key Features <a name="key-features"></a>
 
-Following futures will be added in the next couple days, right now there is no any futures!
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **Addin task**
+- **Removing task**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-> Add a link to your deployed project.
-
 - [Live Demo Link](https://enis-memic.github.io/To-do-list/)
 
```

Comment:
- [ ] [OPTIONAL] 
Kindly also include instructions on how to _run_ or _test_ your app, under the _Getting Started_ section.